packages/uif/src/reducer/types/IndexerAction.ts

```diff
@@ -1,6 +1,7 @@
 export interface InitializedAction {
   type: 'Initialized'
   safeHeight: number
+  configHash?: string
```

Comment:
I understand this will be required in the future? If so, please add a comment here.